FirebaseAuth/Sources/SystemService/FIRSecureTokenService.m

```diff
@@ -200,7 +204,19 @@ - (void)requestAccessToken:(FIRFetchAccessTokenCallback)callback {
 }
```

Comment:
Would it make sense to add another log at around line 190 to track when the token ivars are updated?

Reply:
Done.